backend/inspirehep/serializers.py

```diff
@@ -82,4 +86,18 @@ def serialize_search(self, pid_fetcher, search_result, **kwargs):
             facets. This is used with
             ``inspirehep.search.factories.search.search_factory_only_with_aggs``.
         """
+
+        aggregations = search_result.get("aggregations")
+        new_aggs = deepcopy(aggregations)
+
+        for agg_key, agg_value in aggregations.items():
+            if "nested" in agg_value:
+                nested = agg_value.get("nested")
```

Comment:
you know that there is a `nested` key here so you don't have to use `get` but can use `[`...`]`